www/include/reporting/dashboard/template/viewServicesLog.ihtml

```diff
@@ -1,32 +1,34 @@
 {$form.javascript}
 {$styleB}
 <div class="cachediv">
-<FORM id="datareport">
-	<INPUT type="hidden" name="hostID" value="{$hostID}"><br />
-	<INPUT type="hidden" name="color_OK" value="{$color.OK}">
-	<INPUT type="hidden" name="color_WARNING" value="{$color.WARNING}">
-	<INPUT type="hidden" name="color_CRITICAL" value="{$color.CRITICAL}" >
-	<INPUT type="hidden" name="color_UNKNOWN" value="{$color.UNKNOWN}" >
-	<INPUT type="hidden" name="color_UNDETERMINED" value="{$color.UNDETERMINED}" >
-</FORM>
+	<FORM id="datareport">
```

Comment:
You updated coding style of those lines, so could you also convert type to lower case (FORM --> form, INPUT --> input...)